tests/unit/streamalert_cli/terraform/test_generate.py

```diff
@@ -531,7 +559,7 @@ def test_generate_cloudtrail_cloudwatch_logs_and_s3(self):
                 'prefix': 'unit-test',
                 'region': 'us-west-1',
                 's3_bucket_name': 'unit-test-advanced-streamalert-cloudtrail',
-                's3_logging_bucket': 'unit-test.streamalert.s3-logging',
+                's3_logging_bucket': 'unit-test-streamalert-s3-logging',
```

Comment:
could these be pulled out to `CONSTANTS` so that these can be updated in one place in future?

Reply:
IMHO constants should rarely (never?) be used in unit tests, so going to leave as is